torch/csrc/StorageMethods.cpp

```diff
@@ -49,17 +49,14 @@ static PyObject* THPStorage_nbytes(PyObject* self, PyObject* noargs) {
 
 static PyObject* THPStorage_dataPtr(PyObject* self, PyObject* noargs) {
   HANDLE_TH_ERRORS
-  // PyLong_FromVoidPtr should not need to mutate the pointer in order
```

Comment:
remove the outdated comments introduced by  https://github.com/pytorch/pytorch/pull/97690/files#diff-c6842e062eb6126fcdfc9edf4e9c4fb83849975a221937daca8f5f79395d9900R50